chromium_src/chrome/browser/lifetime/application_lifetime_desktop.cc

```diff
@@ -22,13 +23,13 @@
 namespace chrome {
 
 void AttemptRestart() {
-  if (!brave_relaunch_handler::RelaunchOnMac()) {
+  if (brave::ShouldUseOmaha4() || !brave_relaunch_handler::RelaunchOnMac()) {
```

Comment:
seems like it would be better to add `ShouldUseOmaha4` to `RelaunchOnMac`?

Reply:
Yes, that's nicer. Implemented in ffaea78980fe73697f1f26dc20f6258dcb2c7659.